examples/src/main/scala/sttp/client4/examples/GetRawResponseBodySynchronous.scala

```diff
@@ -13,7 +13,7 @@ object GetRawResponseBodySynchronous extends App {
     .get(uri"https://httpbin.org/get")
     .response(asBoth(asJson[HttpBinResponse], asStringAlways))
 
-  val backend: SyncBackend = HttpClientSyncBackend()
+  val backend: WebSocketSyncBackend = HttpClientSyncBackend()
```

Comment:
I think in the non-websocket examples we could leave `SyncBackend`, since it's a super-trait? We don't need the ws capability here, so maybe it's simpler when it's typed this way?